src/manifest/dbtInstaller.ts

```diff
@@ -0,0 +1,101 @@
+import { window } from 'vscode';
+import { CommandProcessExecution } from '../utils';
+
+enum DBT {
+	Install,
+	Update
+}
+
+enum PromptAnswer {
+	Yes = "Yes",
+	No = "No"
+}
+
+export class DBTClient {
+	private readonly pythonPath: string;
+	static readonly IS_INSTALLED_VERSION = /(?<=installed\sversion:\s)(\d+.\d+.\d+)(?=\D+)/g;
+	static readonly IS_LATESTEST_VERSION = /(?<=latest\sversion:\s)(\d+.\d+.\d+)(?=\D+)/g;
+	static readonly IS_INSTALLED = /installed version/g;
+	static readonly IS_NOT_INSTALLED = /No module named 'dbt'/g;
+
+	constructor(pythonPath: string) {
+		this.pythonPath = pythonPath;
+	}
+
+	public async checkDBTInstalled(): Promise<void> {
+		const checkDBTInstalledProcess = this.checkDBTInstalledCommand();
+		try {
+			await checkDBTInstalledProcess.complete();
+		} catch (err) {
+			if (err.match(DBTClient.IS_NOT_INSTALLED)) {
```

Comment:
This should be a fallback, we should not check for "no module named ...", if we can't find installed version then that means DBT is not installed.